client/src/controllers/scores/scores_controller.js

```diff
@@ -102,6 +109,27 @@ class ScoresController {
 		return badScores;
 	}
 
+	teamIsSelected() {
+		return !this._loading && this._teamNumberList && this._teamNumberList.indexOf(parseInt(this.search)) > -1

```

Comment:
What is the meaning behind `this._teamNumberList.indexOf(parseInt(this.search)) > -1`? What is the logical connection between the question of wether the team is selected to the search mechanism of the scores. And why is this all in the `scores` and not in the `score`?

Reply:
This tests if a team number has been input in the search. This is so it is possible to filter by matches only once a team is selected (as was requested in the issue), as different teams can have different matches.

Reply:
The request was that the match in a score would be editable one the score has a team. The search in this page should search in all fields, with no regards to which exists